dart-runtime/lib/http_client.dart

```diff
@@ -24,6 +24,8 @@ class SdkgenErrorWithData<T> implements Exception {
 
 class SdkgenHttpClient {
   String baseUrl;
+  Map extra;
+  Map<String, String> headers;
```

Comment:
What's the purpose of custom headers?

Reply:
It would be interesting to allow customizing headers for using middleware as API Management that uses a layer above the server to manage APIs.